extras/flux/patch-remove-psp.yaml

```diff
@@ -5,9 +5,17 @@ metadata:
   name: flux-app-pvc-psp-giantswarm
   namespace: flux-giantswarm
 ---
-$patch: delete
-apiVersion: policy/v1beta1
-kind: PodSecurityPolicy
+apiVersion: kustomize.toolkit.fluxcd.io/v1
```

Comment:
For the migration time being, the Customer Flux resources are going to be transferred to the separate Kustomization CR that points to the `bases/flux-app-v2/customer` in the MCB directly, hence we need a patch that removed the PSP from this separate Kustomization CR.